tpm/tpm.go

```diff
@@ -975,6 +975,40 @@ func GetManufacturer(rw io.ReadWriter) ([]byte, error) {
 	return getCapability(rw, capProperty, tpmCapPropManufacturer)
 }
 
+// GetAlgs returns a slice of bytes.
+// A byte is 1 a supported algorithm
```

Comment:
To clarify - each byte is a const identifying an algorithm, right?
Can you define type Algorithm and a bunch of consts for it?